docs/pages/submit/ios.mdx

```diff
@@ -1,99 +1,213 @@
 ---
 title: Submit to the Apple App Store
-sidebar_title: Submit to Apple
 description: Learn how to submit your app to the Apple App Store from your computer and CI.
 ---
 
 import { Collapsible } from '~/ui/components/Collapsible';
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Prerequisites, Requirement } from '~/ui/components/Prerequisites';
 import { Terminal } from '~/ui/components/Snippet';
-import { Step } from '~/ui/components/Step';
 
-This guide outlines how to submit your app to the Apple App Store from your own computer and from CI.
-
-## Prerequisites
-
-- A paid developer account is required to submit an app &mdash; you can create an Apple Developer account on the [Apple Developer Portal](https://developer.apple.com/account/).
-- You will also need to have EAS CLI installed and authenticated with your Expo account:
-  <Terminal cmd={['$ npm install -g eas-cli && eas login']} />
+This guide outlines how to submit your app to the Apple App Store from your computer or from a CI/CD service.
+
+## Submitting your app from your computer
+
+<Prerequisites numberOfRequirements={4}>
+  <Requirement number={1} title="Sign up for an Apple Developer account">
+    An Apple Developer account is required to submit your app to the Apple App Store. You can sign up for an Apple Developer account on the [Apple Developer Portal](https://developer.apple.com/account/).
+  </Requirement>
+  <Requirement number={2} title="Include a bundle identifier in app.json">
+    Include your app's bundle identifier in **app.json**:
+
+    ```json app.json
+    {
+      "ios": {
+        "bundleIdentifier": "com.yourcompany.yourapp"
+      }
+    }
+    ```
+
+  </Requirement>
+  <Requirement number={3} title="Install EAS CLI and authenticate with your Expo account">
+    Install EAS CLI and login with your Expo account:
+    
+    <Terminal cmd={['$ npm install -g eas-cli && eas login']} />
+  </Requirement>
+  <Requirement number={4} title="Build a production app">
+    You'll need a production build ready for store submission. You can create one using [EAS Build](/build/introduction/):
+    
+    <Terminal cmd={['$ eas build --platform ios --profile production']} />
+
+    Alternatively, you can build the app on your own computer with `eas build --platform ios --profile production --local` or with Xcode.
+
+  </Requirement>
+</Prerequisites>
+
+Once you have completed all the prerequisites, you can start the submission process.
+
+Run the following command to submit a build to the Apple App Store:
+
+<Terminal cmd={['$ eas submit --platform ios']} />
+
+The command will lead you step by step through the process of submitting the app. You can configure the submission process by adding a submission profile in **eas.json**:
+
+```json eas.json
+{
+  "submit": {
+    "production": {
+      "ios": {
+        "ascAppId": "your-app-store-connect-app-id"
+      }
+    }
+  }
+}
+```
+
+<Collapsible summary="How to find ascAppId">
```

Comment:
```suggestion
<Collapsible summary={<>How to find <CODE>ascAppId</CODE></>}>
```

Let's update this to use inline syntax highlight. Please add the import statement: `import { CODE } from '~/ui/components/Text';`.